src/controller/python/chip-device-ctrl.py

```diff
@@ -964,15 +965,24 @@ def __format_zcl_arguments_from_dict(optional_args: dict, command: dict) -> Dict
     return formatted_command_args
     
 
-def zcl_add_network(node_id: int, ssid: str, password: str, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, breadcrumb: Optional[int] = 0, timeoutMs: Optional[int] = 1000) -> Dict[str, Any] :
+def zcl_add_network(node_id: int, args: Dict[str, str], endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, breadcrumb: Optional[int] = 0, timeoutMs: Optional[int] = 1000) -> Dict[str, Any] :
```

Comment:
please use Text here instead of str

Reply:
And make this line length shorter please. Should be max 80 chars per line:

```
def zcl_add_network(
    node_id: int, 
    args: Dict[str, str], 
    endpoint_id: Optional[int] = 1, 
    group_id: Optional[int] = 0, 
    breadcrumb: Optional[int] = 0, 
    timeoutMs: Optional[int] = 1000) -> Dict[str, Any] :
```

Reply:
Please also do not use the variable name `args` as `*args` can be reserved for something else in python definitions.

Reply:
add a doc string